src/Microsoft.VisualStudio.ProjectSystem.Managed.VS/ProjectSystem/VS/Tree/Dependencies/GraphNodes/Actions/CheckChildrenGraphActionHandler.cs

```diff
@@ -56,14 +56,10 @@ public override bool HandleRequest(IGraphContext graphContext)
                     continue;
                 }
 
-                if (dependency.Flags.Contains(DependencyTreeFlags.SupportsHierarchy))
-                {
-                    trackChanges = true;
```

Comment:
>trackChanges = true; [](start = 20, length = 20)

Looks like we never set trackChanges to true anymore. Do we still need this variable?